Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -2266,16 +2259,12 @@ def prepare(self, doChecking = False):
         self.pythonCfgCode += self.addCustomise()
 
         if self._options.runUnscheduled:
-            # prune and delete paths
-            #this is not supporting the blacklist at this point since I do not understand it
-            self.pythonCfgCode+="#do not add changes to your config after this point (unless you know what you are doing)\n"
-            self.pythonCfgCode+="from FWCore.ParameterSet.Utilities import convertToUnscheduled\n"
-            self.pythonCfgCode+="process=convertToUnscheduled(process)\n"
-
-            from FWCore.ParameterSet.Utilities import convertToUnscheduled
-            self.process=convertToUnscheduled(self.process)
-
-            self.pythonCfgCode += self.addCustomise(1)
+            print("--runUnscheduled is deprecated and not necessary anymore, and will be removed soon. Please update your command line.")
+        # Keep the "unscheduled customise functions" separate for now,
+        # there are customize functions given by users (in our unit
+        # tests) that need to be run before the "unscheduled customise
+        # functions"
+        self.pythonCfgCode += self.addCustomise(1)
```

Comment:
I tried first to remove the `customisation_file_unsch` altogether, but then found out that (at least) this test
https://github.com/cms-sw/cmssw/blob/master/TauAnalysis/MCEmbeddingTools/test/runtests.sh
depending on customize functions being called before the customize function added with `PAT` step.